docs/docs/sourcing-from-yaml.md

```diff
@@ -0,0 +1,232 @@
+        Cake croissant marshmallow.
+        Sesame snaps apple pie I love I love cake danish powder.
+        Lollipop sweet caramels.
+        Tiramisu danish marshmallow candy canes.
+        Powder cupcake cotton candy bonbon chocolate bar marshmallow gummies cheesecake marzipan.
+        Gummies soufflé candy. Candy canes muffin chocolate brownie pudding.
+  links: []
+- path: "/page4"
+  content:
+    - item:
+        Lorem ipsum dolor amet mustache knausgaard +1, blue bottle waistcoat tbh
+        semiotics artisan synth stumptown gastropub cornhole celiac swag. Brunch raclette
+        vexillologist post-ironic glossier ennui XOXO mlkshk godard pour-over blog tumblr
+        humblebrag. Blue bottle put a bird on it twee prism biodiesel brooklyn. Blue
+        bottle ennui tbh succulents.
+  links:
+    - to: "/page5"
+    - to: "/page1"
+- path: "/page5"
+  content:
+    - item: St. agur blue cheese queso cheesecake.
+        Cheesecake the big cheese monterey jack cheesecake monterey jack paneer halloumi rubber cheese.
+        Cheese triangles cheese strings cheese slices cheesy feet taleggio cottage cheese when the cheese comes out everybody's happy gouda.
+        Feta cauliflower cheese babybel cheese on toast monterey jack.
+    - item:
+        Bacon ipsum dolor amet short ribs brisket venison rump drumstick pig sausage
+        prosciutto chicken spare ribs salami picanha doner. Kevin capicola sausage,
+        buffalo bresaola venison turkey shoulder picanha ham pork tri-tip meatball meatloaf
+        ribeye. Doner spare ribs andouille bacon sausage. Ground round jerky brisket
+        pastrami shank.
+  links:
+    - to: "/page1"
+```
+
+What the code block above is doing, is nothing more, nothing less, than creating a simple YAML object in which:
+
+- Each `path` is a page's endpoint.
+- The `contents` list holds some data to be displayed.
+- The `links` list holds some endpoints to other pages.
+
+## Gatsby configuration
+
+Now that we have almost all the parts necessary in place, you'll need to make a small adjustment to your Gatsby website.
+
+Start by creating a `gatsby-node.js` with the following contents:
+
+```javascript
+const fs = require("fs")
+const yaml = require("js-yaml")
+exports.createPages = ({ actions }) => {
+  const { createPage } = actions
+  const ymlDoc = yaml.safeLoad(fs.readFileSync("./content/index.yaml", "utf-8"))
+  ymlDoc.forEach(element => {
+    createPage({
+      path: element.path,
+      component: require.resolve("./src/templates/basicTemplate.js"),
+      context: {
+        pageContent: element.content,
+        links: element.links,
+      },
+    })
+  })
+}
+```
+
+Breaking down the code into smaller parts:
+
+1. The `js-yaml` package installed earlier is imported.
+2. The `index.yaml` file is loaded and the content parsed.
+3. Through the use of Gatsby's `createPage()` api, the parsed file will be used to create some pages programmatically.
```

Comment:
```suggestion
3. Through the use of Gatsby's [`createPage()` API](/docs/actions/#createPage), the parsed file will be used to create some pages programmatically.
```

Reply:
Changed the phrase according to suggestion